apps/configs/flameshot.ini

```diff
@@ -6,10 +6,10 @@ disabledTrayIcon=true
 drawColor=#ff0000
 drawFontSize=8
 drawThickness=1
-ignoreUpdateToVersion=0.10.2
-savePath=/home/icedborn/Pictures
+ignoreUpdateToVersion=99.99.99
+saveAsFileExtension=.png
```

Comment:
This fixes the name of the config value and changes the extension to png since it's the default suggested by flameshot.